content/04-postgresql/01-benefits-of-postgresql.mdx

```diff
@@ -115,3 +115,42 @@ This focus on community-driven development has resulted in great participation f
 PostgreSQL has gained a great reputation as a powerful, feature-rich choice for relational data.  Valuing stability, functionality, and standards conformance, PostgreSQL checks all of the right boxes for many projects.  Similarly, if you require flexibility in how you can represent data and want to be able to use a variety of tools and languages, PostgreSQL is also a good choice.
 
 PostgreSQL is notable for offering excellent implementation of core relational features while not limiting itself to the boundaries of traditional RDBMSs.  While no database can serve every need, PostgreSQL is an excellent option that is versatile enough to suit many use cases.
+
+## FAQ
+
+<details><summary>Is PostgreSQL open-source?</summary>
+
+[PostgreSQL](https://www.postgresql.org/) is an open-source database. It is an object relational database that has been in development for more than 30 years. 
+
+You can get more acquainted with PostgreSQL in our [Getting to know PostgreSQL feature](https://www.prisma.io/dataguide/postgresql/getting-to-know-postgresql). 
+
+</details>
+
+<details><summary>What is PostgreSQL’s feature matrix?</summary>
+
+[PostgreSQL’s Feature Matrix](postgresql.org/about/featurematrix/) is a table maintained by PostgreSQL outlining all of its features and which version those features were added in. 
+
+You can access each feature’s detailed information by clicking on that feature and getting directed to the appropriate PostgreSQL documentation.
+
+</details>
+
+<details><summary>Where can you find PostgreSQL’s feature list?</summary>
+
+You can find PostgreSQL’s feature list in its [Feature Matrix](postgresql.org/about/featurematrix/).
+
+</details>
+
+<details><summary>Is PostgreSQL free to use?</summary>
+
+PostgreSQL is free to use and is released under the OSI-approved [PostgreSQL License](https://www.postgresql.org/about/licence/). 
+
+There is no fee, even for using commercial software products. 
+
+</details>
+
+<details><summary>Is PostgreSQL a NoSQL database?</summary>
+
+PostgreSQL is an object relational database and is not a NoSQL database.
```

Comment:
I'm not sure if it would be interesting to mention that they have added a few NoSQL features like JSON columns, etc.

Reply:
I like that.